lib/elixir/lib/calendar/datetime.ex

```diff
@@ -245,6 +362,75 @@ defmodule DateTime do
     end
   end
 
+  @doc """
+  Takes a DateTime and a time zone.
+
+  Returns a DateTime for the same point in time, but instead at the
```

Comment:
I would make this line the first one so it becomes the summary

Reply:
I would also include backticks in DateTime and reference the variable (and some rewording):

```
Returns a `DateTime` for the same point in time in the given `time_zone`.
```